main.go

```diff
@@ -547,6 +547,14 @@ func (c *controller) saveHashring(hashring []receive.HashringConfig) error {
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      c.options.configMapGeneratedName,
 			Namespace: c.options.namespace,
+			OwnerReferences: []metav1.OwnerReference{
+				{
+					APIVersion: "v1",
```

Comment:
Should we directly use `orgCM.APIVersion` and `orgCM.Kind`?

Reply:
the current client-go has a problem so that it returns empty if using `orgCM.APIVersion` and `orgCM.Kind`. We need to have another PR to upgrade client-go. Thanks.